install/migrations/update_9.5.x_to_10.0.0/recurrentchange.php

```diff
@@ -83,5 +83,6 @@
 }
 
 $migration->addRight('recurrentchange', ALLSTANDARDRIGHT, [
-    'change' => UPDATE
+    'change' => UPDATE,
```

Comment:
should not it be added in 10.0.2 migration?

Reply:
No, we cannot "downgrade" rights once they have been provided, as people may already have reviewed them.